backend/substrapp/views/filters_utils.py

```diff
@@ -100,6 +102,14 @@ def _get_model_tuple(model):
         raise NotImplementedError
 
 
+def flatten_without_duplicates(list_of_list):
```

Comment:
If speed is important, might be better to do it with a set:

```python
def flatten_set(list_of_list):
    res = set()
    for item in itertools.chain.from_iterable(list_of_list):
        if item not in res:
            res.add(item)
    return list(res)
```

```python
data = [[j for j in range(100)] for i in range(1000)]
%timeit flatten_without_duplicates(data)
%timeit flatten_set(data)
```
--> with list: 10 loops, best of 5: 241 ms per loop
--> with set: 100 loops, best of 5: 8.6 ms per loop

Reply:
I don't know if it's a bottleneck of the application :)

Reply:
Probably not ! Not blocking for the merge